server/common/network/update/council/get.ts

```diff
@@ -0,0 +1,40 @@
+import BigNumber from "bignumber.js";
+import { NextApiRequest, NextApiResponse } from "next";
+
+import models from "db/models";
+
+import { error as LogError } from "services/logging";
+
+export default async function get(req: NextApiRequest, res: NextApiResponse) {
+  try {
+    const { networkId } = req.query;
```

Comment:
we must check if `networkId` is of the type we want, and that it really exists, and if not, throw a 400 error because bad parameters